minkindr_conversions/include/minkindr_conversions/kindr_msg.h

```diff
@@ -17,6 +18,14 @@ void poseKindrToMsg(const kindr::minimal::QuatTransformation& kindr,
                     geometry_msgs::Pose* msg);
 void poseMsgToKindr(const geometry_msgs::Pose& msg,
                     kindr::minimal::QuatTransformation* kindr);
+void poseKindrToMsg(const kindr::minimal::QuatTransformation& kindr,
```

Comment:
poseStampedKindrToMsg? I thought of suggesting that but it doesn't sound right since nothing is stamped in kindr.